--- mate-settings-daemon-1.5.5/./plugins/xrandr/Makefile.am~	2013-02-08 01:25:01.000000000 +0200
+++ mate-settings-daemon-1.5.5/./plugins/xrandr/Makefile.am	2013-02-18 22:09:42.379069593 +0200
@@ -54,7 +54,7 @@
 
 libxrandr_la_CFLAGS =			\
 	$(SETTINGS_PLUGIN_CFLAGS)	\
-	$(LIBMATENOTIFY_CFLAGS)		\
+	$(LIBNOTIFY_CFLAGS)		\
 	$(MATE_DESKTOP_CFLAGS)		\
 	$(AM_CFLAGS)
 
@@ -63,7 +63,7 @@
 
 libxrandr_la_LIBADD  =			\
 	$(SETTINGS_PLUGIN_LIBS)		\
-	$(LIBMATENOTIFY_LIBS)		\
+	$(LIBNOTIFY_LIBS)		\
 	$(MATE_DESKTOP_LIBS)
 
 plugin_in_files =			\
--- mate-settings-daemon-1.5.5/./plugins/xrandr/msd-xrandr-manager.c~	2013-02-08 01:25:01.000000000 +0200
+++ mate-settings-daemon-1.5.5/./plugins/xrandr/msd-xrandr-manager.c	2013-02-18 22:10:12.263775052 +0200
@@ -46,8 +46,8 @@
 #include <libmateui/mate-rr.h>
 #include <libmateui/mate-rr-labeler.h>
 
-#ifdef HAVE_LIBMATENOTIFY
-#include <libmatenotify/notify.h>
+#ifdef HAVE_LIBNOTIFY
+#include <libnotify/notify.h>
 #endif
 
 #include "mate-settings-profile.h"
@@ -1079,7 +1079,7 @@
 static void
 error_message (MsdXrandrManager *mgr, const char *primary_text, GError *error_to_display, const char *secondary_text)
 {
-#ifdef HAVE_LIBMATENOTIFY
+#ifdef HAVE_LIBNOTIFY
         MsdXrandrManagerPrivate *priv = mgr->priv;
         NotifyNotification *notification;
 
@@ -1107,7 +1107,7 @@
 
         gtk_dialog_run (GTK_DIALOG (dialog));
         gtk_widget_destroy (dialog);
-#endif /* HAVE_LIBMATENOTIFY */
+#endif /* HAVE_LIBNOTIFY */
 }
 
 static void
--- mate-settings-daemon-1.5.5/./plugins/a11y-keyboard/Makefile.am~	2013-02-08 01:25:01.000000000 +0200
+++ mate-settings-daemon-1.5.5/./plugins/a11y-keyboard/Makefile.am	2013-02-18 22:10:28.794534045 +0200
@@ -52,7 +52,7 @@
 
 liba11y_keyboard_la_CFLAGS = \
 	$(SETTINGS_PLUGIN_CFLAGS)	\
-	$(LIBMATENOTIFY_CFLAGS)		\
+	$(LIBNOTIFY_CFLAGS)		\
 	$(AM_CFLAGS)
 
 liba11y_keyboard_la_LDFLAGS = 		\
@@ -62,7 +62,7 @@
 liba11y_keyboard_la_LIBADD  = 		\
 	$(SETTINGS_PLUGIN_LIBS)		\
 	$(XF86MISC_LIBS)		\
-	$(LIBMATENOTIFY_LIBS)		\
+	$(LIBNOTIFY_LIBS)		\
 	$(NULL)
 
 plugin_in_files = 		\
--- mate-settings-daemon-1.5.5/./plugins/a11y-keyboard/msd-a11y-keyboard-manager.c~	2013-02-08 01:25:01.000000000 +0200
+++ mate-settings-daemon-1.5.5/./plugins/a11y-keyboard/msd-a11y-keyboard-manager.c	2013-02-18 22:10:59.255932639 +0200
@@ -46,9 +46,9 @@
 #include <X11/extensions/XIproto.h>
 #endif
 
-#ifdef HAVE_LIBMATENOTIFY
-#include <libmatenotify/notify.h>
-#endif /* HAVE_LIBMATENOTIFY */
+#ifdef HAVE_LIBNOTIFY
+#include <libnotify/notify.h>
+#endif /* HAVE_LIBNOTIFY */
 
 #include "mate-settings-profile.h"
 #include "msd-a11y-keyboard-manager.h"
@@ -72,9 +72,9 @@
 
         GSettings  *settings;
 
-#ifdef HAVE_LIBMATENOTIFY
+#ifdef HAVE_LIBNOTIFY
         NotifyNotification *notification;
-#endif /* HAVE_LIBMATENOTIFY */
+#endif /* HAVE_LIBNOTIFY */
 };
 
 static void     msd_a11y_keyboard_manager_class_init  (MsdA11yKeyboardManagerClass *klass);
@@ -497,7 +497,7 @@
         gtk_status_icon_set_visible (manager->priv->status_icon, show);
 }
 
-#ifdef HAVE_LIBMATENOTIFY
+#ifdef HAVE_LIBNOTIFY
 static void
 on_notification_closed (NotifyNotification     *notification,
                         MsdA11yKeyboardManager *manager)
@@ -558,13 +558,13 @@
         }
 }
 
-#endif /* HAVE_LIBMATENOTIFY */
+#endif /* HAVE_LIBNOTIFY */
 
 static gboolean
 ax_slowkeys_warning_post_bubble (MsdA11yKeyboardManager *manager,
                                  gboolean                enabled)
 {
-#ifdef HAVE_LIBMATENOTIFY
+#ifdef HAVE_LIBNOTIFY
         gboolean    res;
         const char *title;
         const char *message;
@@ -625,7 +625,7 @@
         return res;
 #else
         return FALSE;
-#endif /* HAVE_LIBMATENOTIFY */
+#endif /* HAVE_LIBNOTIFY */
 }
 
 
@@ -700,7 +700,7 @@
 ax_stickykeys_warning_post_bubble (MsdA11yKeyboardManager *manager,
                                    gboolean                enabled)
 {
-#ifdef HAVE_LIBMATENOTIFY
+#ifdef HAVE_LIBNOTIFY
         gboolean    res;
         const char *title;
         const char *message;
@@ -764,7 +764,7 @@
         return res;
 #else
         return FALSE;
-#endif /* HAVE_LIBMATENOTIFY */
+#endif /* HAVE_LIBNOTIFY */
 }
 
 static void
@@ -1250,9 +1250,9 @@
 {
         manager->priv = MSD_A11Y_KEYBOARD_MANAGER_GET_PRIVATE (manager);
 
-#ifdef HAVE_LIBMATENOTIFY
+#ifdef HAVE_LIBNOTIFY
         notify_init ("mate-settings-daemon");
-#endif /* HAVE_LIBMATENOTIFY */
+#endif /* HAVE_LIBNOTIFY */
 }
 
 static void
--- mate-settings-daemon-1.5.5/./plugins/housekeeping/Makefile.am~	2013-02-08 01:25:01.000000000 +0200
+++ mate-settings-daemon-1.5.5/./plugins/housekeeping/Makefile.am	2013-02-18 22:11:49.998262432 +0200
@@ -20,12 +20,12 @@
 libhousekeeping_la_CFLAGS = 		\
 	$(SETTINGS_PLUGIN_CFLAGS)	\
 	$(GIOUNIX_CFLAGS)		\
-	$(LIBMATENOTIFY_CFLAGS)		\
+	$(LIBNOTIFY_CFLAGS)		\
 	$(AM_CFLAGS)
 
 libhousekeeping_la_LDFLAGS = $(MSD_PLUGIN_LDFLAGS)
 
-libhousekeeping_la_LIBADD = $(SETTINGS_PLUGIN_LIBS) $(GIOUNIX_LIBS) $(LIBMATENOTIFY_LIBS)
+libhousekeeping_la_LIBADD = $(SETTINGS_PLUGIN_LIBS) $(GIOUNIX_LIBS) $(LIBNOTIFY_LIBS)
 
 plugin_in_files = housekeeping.mate-settings-plugin.in
 
--- mate-settings-daemon-1.5.5/./plugins/housekeeping/msd-disk-space.c~	2013-02-08 01:25:01.000000000 +0200
+++ mate-settings-daemon-1.5.5/./plugins/housekeeping/msd-disk-space.c	2013-02-18 22:12:03.545551115 +0200
@@ -21,7 +21,7 @@
  *
  */
 
-/* gcc -DHAVE_LIBMATENOTIFY -DTEST -Wall `pkg-config --cflags --libs gobject-2.0 gio-unix-2.0 glib-2.0 gtk+-2.0 libmatenotify` -o msd-disk-space-test msd-disk-space.c */
+/* gcc -DHAVE_LIBNOTIFY -DTEST -Wall `pkg-config --cflags --libs gobject-2.0 gio-unix-2.0 glib-2.0 gtk+-2.0 libnotify` -o msd-disk-space-test msd-disk-space.c */
 
 #include "config.h"
 
--- mate-settings-daemon-1.5.5/./configure.ac~	2013-02-08 01:25:01.000000000 +0200
+++ mate-settings-daemon-1.5.5/./configure.ac	2013-02-18 22:13:56.937424046 +0200
@@ -53,7 +53,7 @@
 GTK_REQUIRED_VERSION=2.24.0
 GIO_REQUIRED_VERSION=2.26.0
 MATE_DESKTOP_REQUIRED_VERSION=1.5.0
-LIBMATENOTIFY_REQUIRED_VERSION=1.1.0
+LIBNOTIFY_REQUIRED_VERSION=0.7.5
 
 EXTRA_COMPILE_WARNINGS(yes)
 
@@ -90,24 +90,24 @@
 PKG_CHECK_MODULES(MATE_DESKTOP, mate-desktop-2.0 >= $MATE_DESKTOP_REQUIRED_VERSION)
 
 dnl ---------------------------------------------------------------------------
-dnl - Check for libmatenotify
+dnl - Check for libnotify
 dnl ---------------------------------------------------------------------------
 
-have_libmatenotify=no
-AC_ARG_WITH(libmatenotify,
-	AC_HELP_STRING([--without-libmatenotify], [Disable notifications (default: auto)]),
-	with_libmatenotify=$withval, with_libmatenotify=auto)
-
-if test "x$with_libmatenotify" != "xno"; then
-	PKG_CHECK_MODULES(LIBMATENOTIFY, libmatenotify >= $LIBMATENOTIFY_REQUIRED_VERSION,
-			  [AC_DEFINE(HAVE_LIBMATENOTIFY, 1, [Define if libmatenotify is available])
-			  have_libmatenotify=yes], have_libmatenotify=no)
-	if test "x$have_libmatenotify" = xno -a "x$with_libmatenotify" = xyes; then
-		AC_MSG_ERROR([libmatenotify support requested but libraries not found])
+have_libnotify=no
+AC_ARG_WITH(libnotify,
+	AC_HELP_STRING([--without-libnotify], [Disable notifications (default: auto)]),
+	with_libnotify=$withval, with_libnotify=auto)
+
+if test "x$with_libnotify" != "xno"; then
+	PKG_CHECK_MODULES(LIBNOTIFY, libnotify >= $LIBNOTIFY_REQUIRED_VERSION,
+			  [AC_DEFINE(HAVE_LIBNOTIFY, 1, [Define if libnotify is available])
+			  have_libnotify=yes], have_libnotify=no)
+	if test "x$have_libnotify" = xno -a "x$with_libnotify" = xyes; then
+		AC_MSG_ERROR([libnotify support requested but libraries not found])
 	fi
 fi
-AC_SUBST(LIBMATENOTIFY_CFLAGS)
-AC_SUBST(LIBMATENOTIFY_LIBS)
+AC_SUBST(LIBNOTIFY_CFLAGS)
+AC_SUBST(LIBNOTIFY_LIBS)
 
 dnl ---------------------------------------------------------------------------
 dnl - Check for D-Bus
@@ -520,7 +520,7 @@
     dbus-1 system.d dir:      ${DBUS_SYS_DIR}
     PolicyKit support:        ${HAVE_POLKIT}
 
-    Libmatenotify support:    ${have_libmatenotify}
+    Libnotify support:        ${have_libnotify}
     GStreamer support:        ${have_gstreamer}
     PulseAudio support:       ${have_pulse}
     Smartcard support:        ${have_smartcard_support}
